curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b207e8603d076272706b1.md

```diff
@@ -0,0 +1,58 @@
+---
+id: 657b207e8603d076272706b1
+title: Task 11
+challengeType: 19
+dashedName: task-11
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!-- audio????? -->
+
+# --description--
+
+The suffix `-ly` can be added to words to indicate the frequency of an event. Here's how it applies to different time periods:
+
+- `Daily` means it occurs once every day.
+- `Weekly` indicates something that happens once every week.
+- `Monthly` describes something that happens once every month. 
+- `Yearly` (or annually) describes something that happens once every year.
+
```

Comment:
Are we sure we don't want to use a list here? IMHO it's more readable than as a single paragraph.

Reply:
I personally like keeping this as a list. Found myself skimming through it as a paragraph here versus the original list from earlier

<img width="879" alt="Screenshot 2023-12-18 at 4 59 06 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/fa8f5639-6dd4-4033-ae41-d9fb0e53df25">

Reply:
I removed the list because I thought they shouldn't be here. I personally like the lists, but when I created the challenges the first time they were causing error msgs so I removed them. If you say they are not a problem, we can keep them and in the future I can put them back to other blocks.

Reply:
I'm not getting any errors in the tests or build pipeline.